.github/workflows/windows-build-and-test.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
         build_type: ${{ fromJson(needs.config.outputs.build_type) }}
         ignores: ["chunk_adaptive metadata telemetry"]
         tsl_ignores: ["compression_algos"]
-        tsl_skips: ["bgw_db_scheduler bgw_db_scheduler_fixed"]
+        tsl_skips: ["vector_agg_text vector_agg_groupagg bgw_db_scheduler bgw_db_scheduler_fixed"]
```

Comment:
Why do we need to skip these tests on windows? Is it also because of UMASH?

Reply:
Right, I didn't get it to compile there, so decided to disable for now.